content/pt-br/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md

```diff
@@ -0,0 +1,109 @@
+---
+reviewers:
+- danwent
+- aanm
+title: Utilize o Cilium para NetworkPolicy
+content_type: task
+weight: 30
+---
+
+<!-- overview -->
+Essa página mostra como utilizar o Cilium para NetworkPolicy.
+
+Para saber mais sobre o Cilium, leia o artigo [Introduction to Cilium](https://docs.cilium.io/en/stable/overview/intro).
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+## Fazendo o Deploy do Cilium no Minikube para Testes Básicos
+
+Para familiarizar-se com o Cilium você poderá seguir o guia [Cilium Kubernetes Getting Started Guide](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/) e realizar uma instalação básica do Cilium através de um DaemonSet no minikube.
+
+Inicie o minikube, a versão mínima exigida é >= v1.5.2, com os seguintes argumentos:
+
+```shell
+minikube version
+```
+```console
+minikube version: v1.5.2
+```
+
+```shell
+minikube start --network-plugin=cni
+```
+
+Para o minikube, você poderá instalar o Cilium utilizando a ferramenta de linha de comando (CLI). Para isso, primeiro faça o download da última versão do CLI com o seguinte comando:
+
+```shell
+curl -LO https://github.com/cilium/cilium-cli/releases/latest/download/cilium-linux-amd64.tar.gz
+```
+
+Em seguida extraia o arquivo baixado para o diretório `/usr/local/bin` com os comandos:
+
+```shell
+sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
+rm cilium-linux-amd64.tar.gz
+```
+
+Após executar os passos acima, você poderá instalar o Cilium utilizando o comando abaixo: 
+
+```shell
+cilium install
+```
+
+O Cilium irá detectar as configurações do cluster automaticamente, criará e instalará os componentes apropriados para que a instalação seja bem sucedida.
+Os componentes são:
+
+- Certificate Authority (CA) no Secret `cilium-ca` e os certificados para o Hubble (camada de observabilidade do Cilium).
+- Service accounts.
+- Cluster roles.
+- ConfigMap.
+- Um agente DaemonSet e um Operator Deployment.
+
+Após a instalação, você podera visualizar o status geral do deployment do Cilium com o comando `cilium status`.
+Confira a saída esperada da opção `status` [aqui](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/#validate-the-installation). 
+
+O restante do guia "Getting Started" utiliza como base uma aplicação de exemplo para explicar como aplicar políticas de segurança tanto para L3/L4 (como endereço de IP + porta), quanto para L7 (como HTTP).
```

Comment:
You can translate "Getting Started" to -> ... o restante do guia de primeiros passos 
or something like that.